assets/scripts/info_bubble/RemoveButton.scss

```diff
@@ -7,7 +7,7 @@
   margin-left: 1em;
   font-size: 11px;
 
-  .remove-icon {
+  .octicon {
```

Comment:
is octicon just being used for the trash icon? in which case I'd prefer the other name, or some combo or something more explicit like '.trash-icon'

Also are these width height values still important relative to the 'size' param you are passing in below? I took a quick glance at their code: 
https://github.com/primer/octicons/blob/dbf02d1bb8aa3fc3d3595dc1490dd1962cad0b99/lib/octicons_react/src/get-svg-props.js#L1

So it looks like we'd keep these px sizes if we want the icon to be smaller than 16..but I wonder if we do? The icon _is_ quite small on the gallery and it being larger would make it a slightly bigger target for user to notice and click/tap

Reply:
> is octicon just being used for the trash icon? in which case I'd prefer the other name, or some combo or something more explicit like '.trash-icon'

I agree with you, except that there's no way to add a custom class name to the `<svg>` element they return. We get what we get. Trust me, I already tried that.

> So it looks like we'd keep these px sizes if we want the icon to be smaller than 16..but I wonder if we do? The icon is quite small on the gallery and it being larger would make it a slightly bigger target for user to notice and click/tap

I wasn't quite ready to change the size of the icon itself relative to what we were using before.

Reply:
It occurs to me we might get around that by wrapping the `<svg>` in a `<span>` or something. What do you think - worth the trouble?

Reply:
probably not. could add comment right above that says something like 'this is for the trash can'

Reply:
That sounds fine, I'll add a comment.